liferay/clay
# feat(@clayui/css): LPD-45032 Add new gallery icon

Ticket: https://liferay.atlassian.net/browse/LPD-45032

## Motivation
Add new gallery icon on Clay

## Steps to Verify
1. Fetch this PR and on project/clay, run `yarn site`. 
2. Enter http://localhost:9000/docs/components/icon.html (or the port that is displayed on the terminal)
3. Type "gallery" on search field
4. You can also look for the metaphors `view, media, file, image` and the icon will be displayed.

## Screenshots/videos 
![image](https://github.com/user-attachments/assets/4e9039b9-71f1-40d0-b7d6-eacbf0e5d2d6)
